src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.GetProcessTimes.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        [DllImport(Libraries.Kernel32, SetLastError = true)]
+        internal extern static bool GetProcessTimes(IntPtr handleProcess, out long creation, out long exit, out long kernel, out long user);
```

Comment:
>out long creation, out long exit, out long kernel, out long user [](start = 74, length = 64)

Should the last four parameters be [FILETIME](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.comtypes.filetime?view=netstandard-2.0)?

Reply:
CoreFX seems to be using `long` for this: 

https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Windows/Kernel32/Interop.GetProcessTimes.cs

Reply:
I overlooked at the location of this file :(

---
In reply to: [271986485](https://github.com/dotnet/coreclr/pull/23680#discussion_r271986485) [](ancestors = 271986485)